eppo/README.md

```diff
@@ -0,0 +1,46 @@
+# Agent Check: Eppo / Datadog RUM
+
+## Overview
+
+[Eppo][1] is the experimentation and feature management platform that makes advanced A/B testing accessible to everyone in your organization.
+
+The Datadog Eppo RUM integration enriches your RUM data with your feature flags to provide visibility into performance monitoring and behavioral changes. Determine which users are shown a user experience and if it is negatively affecting the user's performance.
```

Comment:
Will users see this in Datadog or Eppo? Can you clarify what specifically is sent to Datadog?